specs/commandline/sign.md

```diff
@@ -166,6 +167,37 @@ Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da
 notation sign --signature-manifest artifact <registry>/<repository>@<digest>
 ```
 
+### [Preview] Sign artifacts in OCI layout directory on disk
+
+Artifacts can be stored on disk in the form of OCI image Layout defined in spec [OCI image layout][oci-image-layout]. It is a directory structure that contains files and folders. The OCI image layout could be a tarball or a directory in the filesystem. For example, a file named `hello-world.tar` or a directory named `hello-world`. Notation only supports signing OCI layout directory for now. Users can reference an artifact in the layout using either tags, or the exact digest. For example, use `hello-world:v1` or `hello-world@sha256xxx` to reference the container image in OCI layout directory named `hello-world`.
```

Comment:
Currently, we don't support tarball since we don't support `--output <signature_path>`. We can introduce this in the next iteration.

Reply:
> Currently, we don't support tarball since we don't support `--output <signature_path>`. We can introduce this in the next iteration.

@shizhMSFT I understand this. Do you expect me to add this explanation to the spec? I don't think we need to mention this in spec.